### ISSUE TITLE
State of flows

### ISSUE THREAD

**Issue by kibertoad** [taskforcesh/bullmq-pro-support#71]
Documentation says this about Flows:

```md
Flows are a brand new feature in BullMQ, and although is implemented on a stable foundation there could be some unknown issues.
```

Is this still an accurate depiction today? It sounds a bit scary to use it in production if it is implied that it's a beta'ish feature.

**Comment by kibertoad** [comment]
@manast could you please comment on this?

**Comment by manast** [comment]
As far as I know, flows are being used by many users in production, and so far no major issues are reported. I think it is safe to say it is quite stable.

**Comment by kibertoad** [comment]
@manast Thank you! Maybe cautionary line should be removed from the documentation then?

**Comment by manast** [comment]
Yes, I think so. I think it is more to reflect the fact that some advanced cases are still being discussed in the issues, and we still have not agreed on the best way to implement them.
